app/templates/application.hbs

```diff
@@ -1,46 +1,4 @@
-<nav class="navbar navbar-light">
-  <div class="container">
-    <a class="navbar-brand" href="index.html">conduit</a>
-    <ul class="nav navbar-nav pull-xs-right">
-      <li class="nav-item">
-        {{#link-to "home" class="nav-link" }}
-          Home
-        {{/link-to}}
-      </li>
-      {{#if session.isLoggedIn}}
-        <li class="nav-item">
-          {{#link-to "editor.new" class="nav-link" }}
-            <i class="ion-compose"></i>&nbsp;New Article
-          {{/link-to}}
-        </li>
-        <li class="nav-item">
-          {{#link-to "settings" class="nav-link" }}
-            <i class="ion-gear-a"></i>&nbsp;Settings
-          {{/link-to}}
-        </li>
-        <li class="nav-item">
-          {{#link-to "profile" session.user.username class="nav-link" data-test-currentUser-loggedIn=true}}
-            {{#if session.user.image}}
-              <img src={{session.user.image}} class="user-pic" alt="profile image for {{session.user.username}}">
-            {{/if}}
-            {{session.user.username}}
-          {{/link-to}}
-        </li>
-      {{else}}
-        <li class="nav-item">
-          {{#link-to "login" class="nav-link" }}
-            Sign In
-          {{/link-to}}
-        </li>
-        <li class="nav-item">
-          {{#link-to "register" class="nav-link" }}
-            Sign Up
-          {{/link-to}}
-        </li>
-      {{/if}}
-    </ul>
-  </div>
-</nav>
+{{navigation-bar session=session}}
```

Comment:
😍 💅

Reply:
what's the clean up required here? do you want a space after `session=session }}`

Reply:
no clean up, I just liked the fact this had all moved to a component 😬 sorry for the false alarm